vale/WTD/Just.yml

```diff
@@ -0,0 +1,19 @@
+extends: existence
+message: "Consider removing '%s'"
+ignorecase: true
+scope: text
+level: warning
+tokens:
+  - awesome
```

Comment:
Yeah, you've reminded of something I've already realized. A lot of WTD content is more informal blog than docs. I'm tracking here anyway, and will split before merging.